### PR TITLE
Adding interactive example to illustrate checksum and when to use it 

### PR DESCRIPTION
**Pull request by KhaledAbdelgalil**

### CODE REVIEW

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* CRC32_interactive_one.ino
+ * 
+ * This sketch demonstrates when to use CRC checksum 
+ * by playing a beuatiful game with you
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: by playing a beautiful game with you

**Review by EL-SHREIF — COMMENTED** [review]
change the selected word

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ //Serial.println(g_crc32_res);
+}
+
+void loop()
+{
+  
+  if(Serial.available() > 0) {
+      g_incomingByte = Serial.read();
+       Serial.write(g_incomingByte);
+   //if pressed enter
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: remove this

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+void loop()
+{
+  
+  if(Serial.available() > 0) {
+      g_incomingByte = Serial.read();
+       Serial.write(g_incomingByte);
+   //if pressed enter   
+  if((int)g_incomingByte==13)
+  {
+   /* Serial.println("guess:");
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: remove this

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    Serial.println(guess);*/
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: remove this

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+//global variable to store in it reading byte
+char g_incomingByte; 
+//global variable to store result of CRC in it
+uint32_t g_crc32_res;
+//global variable to store your guess
+uint32_t g_guess=0;
+//global variable used for some operations
+uint32_t g_mul=1;
+/**************************************************************************************
+ * SETUP/LOOP
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: SETUP LOOP

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 6]
```diff
@@ -0,0 +1,75 @@
+/* CRC32_interactive_one.ino
+ * 
+ * This sketch demonstrates when to use CRC checksum 
+ * by playing a beuatiful game with you
+ * 
+ * khaled ahmed abdelgalil
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: Remove your name if you still have error

**Review by EL-SHREIF — COMMENTED** [review]
How to solve spell check failure

File: examples/CRC32_interactive_one/CRC32_interactive_one.ino [hunk 7]
```diff
@@ -0,0 +1,71 @@
+/* CRC32_interactive_one.ino
+ * 
+ * This sketch demonstrates when to use CRC checksum 
+ * by playing a beautiful game with you
+ * 
+ * khaled Ahmed
```
Comment (EL-SHREIF) [inline examples/CRC32_interactive_one/CRC32_interactive_one.ino]: matsheel esmak y3m anta bt7bo awy kda leeh????

**Comment by aentinger** [comment]
Thank you @KhaledAbdelgalil for this contribution. However, its not a very good fit for this library which is why I'm closing it.

**Pull request closed without merging**